### PR TITLE
Respond when it can't find anyone to praise

### LINKED ISSUE #4: Add feedback when no action is detected
Add a feature where the bot will reply to the detected mention with some feedback if it does not detect an actionable tweet. 

Something like: 
```
Oops, I don't know who to praise here! 👀 Tweet me with a list of names or the words 'praise me'
```

### PR DESCRIPTION
**Pull request by yorublaireau**
Closes #4

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -50,11 +50,22 @@ def on_status(self, status):
...
       if does_text_says_praise_me(status.text):
           praise(status.user.screen_name)
+          has_praised = True
 
       for mention in get_mentions(status.text, bot_user.screen_name):
           praise(mention)
+          has_praised = True
+
+      if not has_praised:
+        api.update_status(
```
Comment (yorublaireau) [inline app.py:64]: I can't really test that so it's a best guess for the api

**Comment by yorublaireau** [comment]
I still need to update the changelog, but the PR isn't merged yet, so I'll do that once #12 is merged in

File: app.py (hunk 1 above)
Comment (rwnx) [reply app.py]: Move fast and break things, baby!

**Comment by rwnx** [comment]
https://twitter.com/praiserbot/status/1577606111712116738

**Pull request merged**